aion/web/elm/src/Room/Notifications.elm

```diff
@@ -17,16 +17,39 @@ addToast toast ( model, cmd ) =
     Toasty.addToast myConfig ToastyMsg toast ( model, cmd )
 
 
+
+-- answer feedback toasts
+
+
 incorrectAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 incorrectAnswerToast =
-    addToast (Toasty.Defaults.Error "Error!" "Wrong Answer")
+    addToast (Toasty.Defaults.Error "Error!" "Wrong answer!")
 
 
 closeAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 closeAnswerToast =
-    addToast (Toasty.Defaults.Warning "Close one!" "Your Answer is Almost Correct")
+    addToast (Toasty.Defaults.Warning "Close one!" "Your answer is almost correct!")
 
 
 correctAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 correctAnswerToast =
-    addToast (Toasty.Defaults.Success "Good Answer!" "Your Answer is Correct")
+    addToast (Toasty.Defaults.Success "Good Answer!" "Your answer is correct!")
+
+
+
+-- question creation
+
+
+questionFormValidationErrorToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
+questionFormValidationErrorToast =
+    addToast (Toasty.Defaults.Error "Error!" "Submission form is not valid!")
+
+
+questionCreationErrorToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
+questionCreationErrorToast =
+    addToast (Toasty.Defaults.Error "Error!" "Failed to create a question.")
+
+
+questionCreationSuccessfulToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
+questionCreationSuccessfulToast =
+    addToast (Toasty.Defaults.Success "Success!" "Question created successfully.")
```

Comment:
I have no idea yet. I suggested my approach to that one, I think it would be cool to have Notifications.elm file containing only notification messages and have different notification files in certain resources.